content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -133,6 +133,39 @@ log audit backend using the following [kube-apiserver][kube-apiserver] flags:
 - `--audit-log-maxbackup` defines the maximum number of audit log files to retain
 - `--audit-log-maxsize` defines the maximum size in megabytes of the audit log file before it gets rotated
 
+{{< note >}}
```

Comment:
Please consider removing the `note` shortcode, because this "note" is pretty complicated and verbose.
Sometimes we see strange behaviors when nesting things.